projects/vdk-core/src/vdk/internal/builtin_plugins/termination_message/writer.py

```diff
@@ -29,9 +29,10 @@ def vdk_configure(self, config_builder: ConfigurationBuilder):
 
     @hookimpl
     def vdk_exit(self, context: CoreContext, exit_code: int):
+        not_resolved_predicate = lambda e: not e.resolved
         self.write_termination_message(
-            errors.get_blamee_overall(),  # TODO: get this from context
-            errors.get_blamee_overall_user_error(),  # TODO: get this from context
+            errors.get_blamee_overall(not_resolved_predicate),
```

Comment:
To be it makes intuitive sense to filter based on the not resolved errors.
Why is it in the file 
`projects/control-service/versatile-data-kit/projects/vdk-core/src/vdk/internal/builtin_plugins/notification/notification.py`. 
that you would include resolved errors too?